llvm/test/Assembler/amdgcn-unreachable.ll

```diff
@@ -0,0 +1,21 @@
+; RUN: llvm-as < %s | llvm-dis | FileCheck %s
```

Comment:
Don't we have `llvm/test/Verifier/AMDGPU` dedicated for our own IR verifier tests?

Reply:
test/Verifier is for failing cases that produce an error, not valid cases that do not